services/main-frontend/src/components/page-specific/organizations/index/CreateOrganizationPopup.tsx

```diff
@@ -0,0 +1,144 @@
+import { css } from "@emotion/css"
+import React from "react"
+import { useTranslation } from "react-i18next"
+
+import StandardDialog from "@/shared-module/common/components/dialogs/StandardDialog"
+
+interface CreateOrganizationPopupProps {
+  show: boolean
+  setShow: React.Dispatch<React.SetStateAction<boolean>>
+  name: string
+  setName: React.Dispatch<React.SetStateAction<string>>
+  visibility: string
+  setVisibility: React.Dispatch<React.SetStateAction<string>>
+  handleCreate: () => void
+  slug: string
+  setSlug: React.Dispatch<React.SetStateAction<string>>
+}
+
+const CreateOrganizationPopup: React.FC<CreateOrganizationPopupProps> = ({
+  show,
+  setShow,
+  name,
+  setName,
+  visibility,
+  setVisibility,
+  handleCreate,
+  slug,
+  setSlug,
+}) => {
+  const { t } = useTranslation("main-frontend") as { t: (key: string) => string }
+
+  return (
+    <StandardDialog
+      open={show}
+      onClose={() => setShow(false)}
+      title={t("create-organization-title")}
+      buttons={[
+        {
+          children: t("create"),
+          onClick: handleCreate,
+
+          variant: "primary",
+        },
+        {
+          children: t("button-text-cancel"),
+          onClick: () => setShow(false),
+
+          variant: "secondary",
+        },
+      ]}
+    >
+      <p
+        className={css`
+          font-size: 16px;
+          margin-bottom: 32px;
+        `}
+      >
+        {t("create-organization-description")}
+      </p>
+
+      <div
+        className={css`
+          display: flex;
+          flex-direction: column;
+          gap: 24px;
+          margin-bottom: 32px;
+        `}
+      >
+        {/* Organization Name */}
+        <div
+          className={css`
+            flex: 1;
+            display: flex;
+            flex-direction: column;
+          `}
+        >
+          <label htmlFor="org-name">{t("label-organization-name")}</label>
```

Comment:
I know that the design had a custom design for the text fields, but since we have custom components for input fields, maybe it would be anyway better if we just used those. Also I would use react-hook-form for the form fields, and maybe also handle the mutation in this component too. (That way we don't have to pass so much stuff to the parent. Now if we added new fields to the form, we would have to add a new prop for each new form field)